aldryn_jobs/cms_wizards.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+from django import forms
+
+from cms.api import add_plugin
+from cms.utils import permissions
+from cms.wizards.wizard_pool import wizard_pool
+from cms.wizards.wizard_base import Wizard
+from cms.wizards.forms import BaseFormMixin
+
+from parler.forms import TranslatableModelForm
+
+from .cms_appconfig import JobsConfig
+from .models import JobCategory, JobOpening
+from .utils import namespace_is_apphooked
+
+
+class ConfigCheckMixin(object):
+
+    def user_has_add_permission(self, user, **kwargs):
+        """
+        Return True if the current user has permission to add a JobCategory or
+        JobOpening (depending on value of `perm_string` class variable.
+        :param user: The current user
+        :param kwargs: Ignored here
+        :return: True if user has add permission, else False
+        """
+        # No one can create an Article, if there is no app_config yet.
+        configs = JobsConfig.objects.all()
+        if not configs:
+            return False
+        if not any([namespace_is_apphooked(config.namespace)
+                    for config in configs]):
+            return False
```

Comment:
These two conditions could be combined, I think. They're both very related and have the same effect.
`if not configs or not any(...):`